jobs/build/promote/Jenkinsfile

```diff
@@ -228,15 +230,19 @@ node {
         if (!params.VERSION) {
             error("You must explicitly specify a VERSION to promote a release for a non-stream assembly.")
         }
+        if (params.ADVISORY) {
+            error("Cannot override image advisory number for a non-stream assembly.")
+        }
         (major, minor) = commonlib.extractMajorMinorVersionNumbers(params.VERSION)
         group = "openshift-${major}.${minor}"
         // FIXME: Only support standard X.Y.Z and hotfix releases for now. RCs and FCs are not supported yet.
         release_config = buildlib.get_releases_config(group)?.releases?.get(params.ASSEMBLY)
         if (!release_config) {
             error("Assembly ${params.ASSEMBLY} is not defined in releases.yml.")
         }
-
-        direct_release_nightly = !release.getAdvisories(group)?.image  // Whether this assembly should go through Errata process is determined by group config advisories.
+        advisory_map = release.getAdvisories(group)?: [:]
+        advisory = advisory_map?.image?: -1
+        direct_release_nightly = advisory <= 0  // Whether this assembly should go through Errata process is determined by group config advisories.
```

Comment:
if we're talking about assemblies what do the group config advisories have to do with it?

Reply:
Group config already has assembly group config merged on top of group.yml: https://github.com/openshift/doozer/blob/633736b5fd852bb4d6e82856f55291e03594789d/doozerlib/runtime.py#L304